app/code/Magento/ImportExport/view/adminhtml/templates/import/form/before.phtml

```diff
@@ -209,7 +209,7 @@ require([
         postToFrameProcessResponse: function(response) {
             if ('object' != typeof(response)) {
                 alert({
-                    content: 'Invalid response'
+                    content: '<?= $block->escapeHtml(__('Invalid response')); ?>'
```

Comment:
Please, use `mage/translation` instead of block method

Reply:
I don't know the reason but the first time of this PR, I use the mage/translate. And it can not translate. Could you please try it?

![image](https://user-images.githubusercontent.com/8234209/63036620-93542980-bee7-11e9-8da2-c2ee406802c8.png)

Reply:
`escapeHtml` method does not translate strings, it just removes special characters from the string.

Reply:
@sidolov : Yes, but __(‘...’) translate the string. And the code now can translate. I use escapeHtml because I make it similar many place in Magento now. It is strange because the code escapeHtml of ‘fixed’ string. I can remove that code if you want. 
I mean the mage/translate can not translate in this case (I don’t know but I may be a bug of mage/translate) . So I use translate by ‘block’.

Reply:
@sidolov :I think escapeHtml to Secure the change of csv file can include the “html” code. So it is the reason escapeHtml of __(‘...’) function.

Reply:
@edenduong looks like @engcom-Alfa didn't clear the cache after changes were applied. `mage/translate` works similar to __() function on the backend. Please, revert back changes to use `mage/translate` and I'll accept your PR. I'm sorry for the inconvenience.

Reply:
@sidolov : I don't know the reason but `mage/translate` can not translate. (I have cleared cache). I tested it. Could you please try to edit it?

Reply:
@edenduong you are right looks like `mage/translate` does not work in phtml file, it's working on js only. Thank you for the clarification!